src/components/TimeseriesChart/CursorOverview.tsx

```diff
@@ -132,25 +132,31 @@ export class CursorOverview extends React.Component<
       return Math.max(rulerPoint.timestamp, acc);
     }, 0);
 
-    const timeLabel =
-      typeof ruler.timeLabel === 'function'
-        ? ruler.timeLabel({
-            id: 0,
-            name: '',
-            value: 0,
-            color: '#ccddff',
-            timestamp: newestTimestamp,
-            x: 0,
-            y: 0,
-          })
-        : formattedDate(newestTimestamp);
-
-    const yLabelFn: (point: ChartRulerPoint) => string =
-      typeof ruler.yLabel === 'function'
-        ? ruler.yLabel
-        : (point: ChartRulerPoint) => {
-            return numeral(point.value).format('0[.]0[00] a');
-          };
+    const timeLabel = ruler.timeLabel
+      ? ruler.timeLabel({
+          id: 0,
+          name: '',
+          value: 0,
+          color: '#ccddff',
+          timestamp: newestTimestamp,
+          x: 0,
+          y: 0,
+        })
+      : formattedDate(newestTimestamp);
+
+    const yLabelFn: (point: ChartRulerPoint) => string = ruler.yLabel
+      ? ruler.yLabel
+      : (point: ChartRulerPoint) => {
+          return numeral(point.value).format('0[.]0[00] a');
+        };
+
+    const renderTag = (s: any) =>
```

Comment:
please use better variable name and destruct object parameter

Reply:
Fixed with `const renderTag = ( {id, color}: {id: number, color: string} ) =>`